exercises/ledger/canonical-data.json

```
@@ -0,0 +1,240 @@
+{
+  "exercise": "ledger",
+  "cases": [
+    {
+      "uuid": "d131ecae-a30e-436c-b8f3-858039a27234",
+      "description": "empty ledger",
+      "property": "formatEntries",
+      "input": {
+        "currency": "USD",
+        "locale": "en_US",
+        "entries": []
+      },
+      "expected": ["Date       | Description               | Change       "]
+    },
+    {
+      "uuid": "ce4618d2-9379-4eca-b207-9df1c4ec8aaa",
+      "description": "one entry",
+      "property": "formatEntries",
+      "input": {
+        "currency": "USD",
+        "locale": "en_US",
+        "entries": [
+          {
+            "date": "2015-01-01",
+            "description": "Buy present",
+            "amount": -1000
```

Maybe add the denomination?

```suggestion
            "amountInCents": -1000
```